### DISCUSSION TITLE
1.1.9

### DISCUSSION THREAD

**Discussion by coreybutler** [coreybutler/nvm-windows#705] [General]
## What's Changed
* **Code signed release**.
* Fix broken link to nodejs versions by @crhistianramirez in https://github.com/coreybutler/nvm-windows/pull/679
* Fix readme links. by @veleek in https://github.com/coreybutler/nvm-windows/pull/689
* docs: update address of china mirror by @hyj1991 in https://github.com/coreybutler/nvm-windows/pull/692
* Use Windows defined env vars in all paths in README by @gdziadkiewicz in https://github.com/coreybutler/nvm-windows/pull/699

## New Contributors
* @crhistianramirez made their first contribution in https://github.com/coreybutler/nvm-windows/pull/679
* @veleek made their first contribution in https://github.com/coreybutler/nvm-windows/pull/689
* @hyj1991 made their first contribution in https://github.com/coreybutler/nvm-windows/pull/692
* @gdziadkiewicz made their first contribution in https://github.com/coreybutler/nvm-windows/pull/699

**Full Changelog**: https://github.com/coreybutler/nvm-windows/compare/1.1.8...1.1.9

<hr /><em>This discussion was created from the release <a href='https://github.com/coreybutler/nvm-windows/releases/tag/1.1.9'>1.1.9</a>.</em>

**Comment by sushruth** [comment]
Hi @coreybutler, Can you please consider making the `nvm-setup.exe` file directly available as a release artifact instead of the zip file?

I wanted to submit a `winget` manifest for `nvm-windows` if that is possible so that nvm-windows can be installed via winget like this - 

```sh
winget install -e --id nvm-windows
```

Please share your thoughts on that. Thanks.

**Reply by coreybutler** [reply]
@sushruth - [done](https://github.com/coreybutler/nvm-windows/releases/tag/1.1.9). When you're finished, let me know. I'll add something to the README (or feel free to submit a PR to update the README).

To date, it has only been available as a zip file because some people are blocked from directly downloading exe files (and nobody has asked for it directly until now). I've added the exe to the existing 1.1.9 release.

**Reply by awdr74100** [reply]
Hi @coreybutler, thank you for providing such a useful tool, and I have submitted this to winget, download now:

```sh
winget install -e --id CoreyButler.NVMforWindows
```

But there seem to be some problem:

![螢幕擷取畫面 2022-05-04 005358](https://user-images.githubusercontent.com/47501692/166501746-b8ade2c1-832a-4ffc-a413-d09a4d86ff1b.png)

The system doesn't seem to recognize the version number, may be a problem with writing to the registry? refer to this [discussion](https://github.com/microsoft/winget-pkgs/issues/16003).

Please help solve the problem. Thanks!

**Reply by coreybutler** [reply]
I've never worked with winget, so I'm not sure how much help I can be. What I do know is the version number is [defined in the installer](https://github.com/coreybutler/nvm-windows/blob/master/nvm.iss#L4). The screenshot above is showing the version (1.1.9)... so I don't know what the problem is. Keep in mind, I only speak English and a little German, so I don't entirely understand the labels in the screenshot.

**Reply by awdr74100** [reply]
@coreybutler check this #796. It would be great if it could.

**Reply by sushruth** [reply]
@coreybutler , @awdr74100 - I was able to succesfully install nvm now - 

<img width="564" alt="image" src="https://user-images.githubusercontent.com/2114206/167020471-e883b9ed-5d31-445c-a121-53b5dbdd277d.png">

command - `winget install -e --silent --id CoreyButler.NVMforWindows`

**Reply by awdr74100** [reply]
@sushruth enjoy! But now the problem is that **DisplayVersion** has not been correctly written into the registry, leading to the appearance of **Unknown** version. Using `winget upgrade --all` it will be updated continuously. I have submitted #796 for review.

**Reply by massimo1220** [reply]
it's good.

**Reply by ticktack** [reply]
very nice.

**Comment by raatchaapol** [comment]
I've installed the 1.1.9 version, but why is my NVM-Windows still the 1.1.7 version?
